internal/manifests/persistentvolume/persistentvolumeclaim.go

```diff
@@ -74,6 +59,26 @@ func CreateOrUpdatePVC(ctx context.Context, c client.Client, pvc *corev1.Persist
 	return nil
 }
 
+// CreateOrUpdatePVC attempts first to create the given persistentvolumeclaim. If the
+// persistentvolumeclaim already exists and the provided comparison func detects any changes
+// an update is attempted. Updates are retried with backoff (See retry.DefaultRetry).
+// Returns on failure an non-nil error.
+func CreateOrUpdatePVC(ctx context.Context, c client.Client, pvc *corev1.PersistentVolumeClaim, equal EqualityPVCFunc, mutate MutatePVCFunc) error {
+	err := c.Create(ctx, pvc)
```

Comment:
I just changed this a function in the CLO and this pattern is pervasive in both our operators but it is backwords.  The majority of the time we will get/compare/update with less frequently creating.  This should be the default pattern:
```
get
if not found
    create
else
   if compare different
      update
```

Reply:
I see. I think after a refactor on our end, a number of CreateOrUpdate functions got turned into

```
create
get
if compare different
   update
```

Changing it to that in the original function, should just clean up your previous options so I'll just go for the default pattern and then write a card to evaluate how many of our functions need to have this change in the pattern done as well.